.github/workflows/build-plugin-zip.yml

```diff
@@ -18,53 +18,20 @@ concurrency:
     cancel-in-progress: true
 
 jobs:
-    compute-stable-branches:
-        name: Compute current and next stable release branches
-        runs-on: ubuntu-latest
-        outputs:
-            current_stable_branch: ${{ steps.get_branches.outputs.current_stable_branch }}
-            next_stable_branch: ${{ steps.get_branches.outputs.next_stable_branch }}
-
-        steps:
-            - name: Get current and next stable release branches
-              id: get_branches
-              run: |
-                  curl \
-                    -H "Accept: application/vnd.github.v3+json" \
-                    -o latest.json \
-                    "https://api.github.com/repos/${{ github.repository }}/releases/latest"
-                  LATEST_STABLE_TAG=$(jq --raw-output '.tag_name' latest.json)
-                  IFS='.' read LATEST_STABLE_MAJOR LATEST_STABLE_MINOR LATEST_STABLE_PATCH <<< "${LATEST_STABLE_TAG#v}"
-                  echo "::set-output name=current_stable_branch::release/${LATEST_STABLE_MAJOR}.${LATEST_STABLE_MINOR}"
-                  if [[ ${LATEST_STABLE_MINOR} == "9" ]]; then
-                    echo "::set-output name=next_stable_branch::release/$((LATEST_STABLE_MAJOR + 1)).0"
-                  else
-                    echo "::set-output name=next_stable_branch::release/${LATEST_STABLE_MAJOR}.$((LATEST_STABLE_MINOR + 1))"
-                  fi
-
```

Comment:
Removing this would completely break our [release tooling](https://github.com/WordPress/gutenberg/blob/cba842a101df8a0cc0c2476768facc0c1e017532/docs/contributors/code/release.md#release-tool). (Same for a number of other changes below.) I'm guessing this wasn't deliberate but the result of a rebase that didn't go quite right? 😅

Reply:
100% — this branch was quite behind, I'll put everything back 😅